src/blockchain/node/filteroracle/BlockIndexer.hpp

```diff
@@ -131,7 +131,7 @@ class FilterOracle::BlockIndexer final : public BlockDMFilter,
     ~BlockIndexer() final;
 
 protected:
-    auto pipeline(zmq::Message&& in) noexcept -> void final;
+    auto pipeline(zmq::Message&& in) -> void override final;
```

Comment:
I would not be doing that, as final implies override. Some people consider this bad style.